Purchases/Public/Errors/ErrorCode.swift

```diff
@@ -131,7 +131,7 @@ extension ErrorCode: DescribableError {
         case .emptySubscriberAttributes:
             return "A request for subscriber attributes returned none."
         case .productDiscountMissingIdentifierError:
-            return "SKProductDiscount must have a non-empty identifier. This is possibly an App Store quirk."
+            return "StoreProductDiscount must have a non-empty identifier. This is possibly an App Store quirk."
```

Comment:
This message probably should change... The empty-identifier is a StoreKit thing that is out of our control while the StoreProductDiscount is in our control.
So maybe something like 
```
The SKProductDiscount wrapped by the StoreProductDiscount is missing an identifier. 
This is a required property and likely an AppStore quirk that it is missing.
```

Reply:
This will either be `SKProductDiscount` or `Product.SubscriptionOffer` so...

```
The SKProductDiscount or `Product.SubscriptionOffer` wrapped by the StoreProductDiscount is missing an identifier. 
This is a required property and likely an AppStore quirk that it is missing.
```

Does that work? 🤔

Reply:
🚀🐐